### PR TITLE
FISH-9803 - NPE prevented by update of index in array to prevent null value as input.

### PR DESCRIPTION
**Pull request by adLuk**
…ameter.

### CODE REVIEW

**Review by simonladen — APPROVED** [review]

File: appserver/webservices/jsr109-impl/src/main/java/org/glassfish/webservices/monitoring/WebServiceTesterServlet.java [hunk 1]
```diff
@@ -538,7 +538,7 @@ private String wsImport(URL wsdlLocation) throws IOException {
             logger.log(Level.SEVERE, LogUtils.CREATE_DIR_FAILED, classesDir);
         }
 
-        String[] wsimportArgs = new String[14];
+        String[] wsimportArgs = new String[13];
```
Comment (svendiedrichsen) [inline appserver/webservices/jsr109-impl/src/main/java/org/glassfish/webservices/monitoring/WebServiceTesterServlet.java:541]: Why not use a List<String> here? Why bother with indexes?

Comment (adLuk) [reply appserver/webservices/jsr109-impl/src/main/java/org/glassfish/webservices/monitoring/WebServiceTesterServlet.java]: To be honest I was trying to keep changes as small as possible. Just to fix problem not really improve code. But I will do it, also I can open one more issue related to this class as technical depth and try to at least remove some of these potecially problematic parts of code.

**Comment by aubi** [comment]
The new version is fine, waiting for CLA.

**Comment by smillidge** [comment]
CLA received last week

**Pull request merged**